### ISSUE TITLE
Allow alternative references

### ISSUE THREAD

**Issue by Bernadetadad** [dms-vep/dms-vep-pipeline-3#47]
@jbloom For H5 numbering I think ideally I can have both H3 and H1 numbering. 
Current pipeline I think only allows for `sequential_site` and `reference_site` and it would be good to have something like `reference_site_h3` / `reference_site_h1` allowed.

**Comment by Bernadetadad** [comment]
`[site_numbering_map.csv](https://github.com/dms-vep/dms-vep-pipeline-3/files/12402717/site_numbering_map.csv)` with alternative reference sites att.

**Comment by jbloom** [comment]
@Bernadetadad, just looking at this file, it is wrong. You have rows such as:
```
sequential_site,reference_site_H3,reference_site_H1,region
1,NaN,NaN,HA1
2,NaN,NaN,HA1
```

A site number cannot be `NaN`. There must be a one-to-one mapping between all site numbers and they must all be defined. If a site is an insertion, it is defined like `100a`. If it is before one, they are often defined as `-2`, `-1`, etc. But the sites are part of the protein, and each site must be given a unique label in each numbering scheme.

**Comment by jbloom** [comment]
@Bernadetadad, this is now implemented in version 3.2.4.

Any column in `site_numbering_map.csv` that ends in "site" will now be shown as a tooltip. You also no longer need to specify: `addtl_tooltip_stats: ["sequential site"]` in `antibody_escape_config.yaml`, so you can remove that.

We still need to update this in the overall H5 summary plot made outside the pipeline in `custom_rules.smk`.

If you let me know when you have updated the H5 HA repo to use this newest version, I can add that if you want.
